scripts/list_unused_topics.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+CURRENT_DIR="$( pwd -P)"
+SCRIPT_SRC="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P )"
+TOPICS_SRC="$( dirname $SCRIPT_SRC )/docs/topics"
+DOCS_SRC="$( dirname $SCRIPT_SRC )/docs"
+SKIP="$SCRIPT_SRC/skip_unused_files.txt"
+
+cd $DOCS_SRC
```

Comment:
I recommend using `pushd` instead of `cd` everywhere because if you interrupt the script halfway through, you'll automatically be returned to the previous directory in the directory stack, i. e. your original working directory. (I may be wrong here, but I think it works this way)

Reply:
@tradej FYI, `pushd` causes 3 lines of output at the beginning that are relative paths to the search dirs:

```bash
~/github/documentation/docs ~/github/documentation
~/github/documentation/docs/topics ~/github/documentation/docs ~/github/documentation
~/github/documentation/docs ~/github/documentation/docs/topics ~/github/documentation/docs ~/github/documentation
```

I'll see if i can work around this. 

Otherwise, output is identical.

Reply:
found a workaround.